package auth.sys.exception;

public class UserAlreadyExistsException {
}
